scripts/gm/bin/lib-gm

```diff
@@ -692,10 +692,12 @@ list_keys() {
 }
 
 hermes_config() {
+  STRATEGY="$(stoml -q "$GLOBAL_HERMES_CONFIG" global.strategy || echo 'all')"
+  LOG_LEVEL="$(stoml -q "$GLOBAL_HERMES_CONFIG" global.log_level || echo 'info')"
```

Comment:
I would like to keep all config reading in the `load_config` function and make variables available for functions from there. Can you move this in there? You'll find a section that loads all global configs.

I would also rename them `GLOBAL_HERMES_STRATEGY` and `GLOBAL_HERMES_LOG_LEVEL` as well as the option to `global.hermes_strategy` and `global.hermes_log_level`, as gaiad might have log levels in the future.

I guess soon we'll need to introduce a [hermes] global section in the config and move all entries there so the names don't get too long. At that point we can name them `hermes.log_level`.

Also note that the default loading (`||`) is not fully working, there's an issue about it in #993. Don't worry about it for now, just make sure you have these options in your config so you don't rely on hard-coded defaults.

Reply:
> I would like to keep all config reading in the load_config function and make variables available for functions from there. Can you move this in there? You'll find a section that loads all global configs.

That makes sense. Should be fixed now. 2d033ab

> I would also rename them GLOBAL_HERMES_STRATEGY and GLOBAL_HERMES_LOG_LEVEL as well as the option to global.hermes_strategy and global.hermes_log_level, as gaiad might have log levels in the future.
> I guess soon we'll need to introduce a [hermes] global section in the config and move all entries there so the names don't get too long. At that point we can name them hermes.log_level.

I renamed the variables into `GLOBAL_HERMES_STRATEGY` and `GLOBAL_HERMES_LOG_LEVEL` but I think we should postpone changing the Hermes config.toml file to adopt `global.hermes_strategy` and `global.hermes_log_level` (that would require many changes in the `ibc-relayer` crate).

> I guess soon we'll need to introduce a [hermes] global section in the config and move all entries there so the names don't get too long. At that point we can name them hermes.log_level.

Agree, we'll do this later.